### PR TITLE
887 integrate lint in flow

### PR DESCRIPTION
**Pull request by ShiriMoran**
First stage of integrating lint #891 - separating the execution to analysis and print

### CODE REVIEW

File: pkg/linter/linterExecute.go [hunk 1]
```diff
@@ -84,14 +94,20 @@ func LinterExecute(configs map[string]*vpcmodel.VPCConfig, printAllFindings bool
...
+			return nil, err
 		}
+	}
+	return linters, nil
+}
+
+func (linters Linters) String(printAllFindings bool) (resString string) {
+	strPerLint := []string{}
+	for _, thisLinter := range linters {
+		thisLintStr := ""
```
Comment (zivnevo) [inline pkg/linter/linterExecute.go:108]:
```suggestion
```
Redundant line

File: pkg/linter/linterExecute.go [hunk 2]
```diff
@@ -65,17 +65,27 @@ func computeConnectivity(configs map[string]*vpcmodel.VPCConfig) (map[string]*vp
...
+// integrated in the general flow
 func LinterExecute(configs map[string]*vpcmodel.VPCConfig, printAllFindings bool,
 	enableList, disableList []string) (issueFound bool, resString string, err error) {
-	nodesConn, err := computeConnectivity(configs)
+	linters, err := linterAnalysis(configs, enableList, disableList)
 	if err != nil {
 		return false, "", err
 	}
+	resString = linters.String(printAllFindings)
+	fmt.Println(resString)
```
Comment (zivnevo) [inline pkg/linter/linterExecute.go:77]: `linters.String()` also has a `Println` for the results - so results are printed twice?

File: pkg/linter/linterExecute.go [hunk 3]
```diff
@@ -65,17 +65,27 @@ func computeConnectivity(configs map[string]*vpcmodel.VPCConfig) (map[string]*vp
...
+	return issueFound, resString, nil
```
Comment (zivnevo) [inline pkg/linter/linterExecute.go:78]: Where do you set `issuesFound`?

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/linter/linterExecute.go (hunk 3 above)
Comment (ShiriMoran) [reply pkg/linter/linterExecute.go]: its not, and not really need. removed (the function was supposed to be deleted before PR I did not bother in the first place)

File: pkg/linter/linterExecute.go (hunk 2 above)
Comment (ShiriMoran) [reply pkg/linter/linterExecute.go:77]: I removed the `Println` from `linters.String()`
I still believe  `LinterExecute` will be deleted soon

File: pkg/linter/linterExecute.go (hunk 1 above)
Comment (ShiriMoran) [reply pkg/linter/linterExecute.go]: right. removed (and replace = in  := where needed)

**Review by zivnevo — APPROVED** [review]

**Pull request merged**